build_raven

```diff
@@ -87,6 +87,8 @@ else
   echo No Conda Python libraries ...
 fi
 
+echo Installing ExamplePlugin ...
+./scripts/install_plugins.py -s ./plugins/ExamplePlugin
```

Comment:
I don't know that it matters yet for the example plugin, but generally plugins need to be identified before `raven_libraries` is built, otherwise the environment will be missing plugin libraries.

I imagine we'd want to test this through the example plugin, but maybe not?

Reply:
That's a good point. I'm not sure what "test" library would be good to require in ExamplePlugin; but I can move this portion of code to before the environment build portion.

Reply:
Unfortunately that code block you moved it before isn't the library installation, that's just loading the libraries. The library installation is a separate installation step before the `build_raven` script is run.

If we had an all-inclusive installer that ran plugin install, then library building, then raven building, then we could make sure the example is always installed. 

Right now in the installation procedure we expect the user to install plugins using the "-a" option, which automatically grabs the example plugin as well as the other standard plugins.